pkg/kotsadm/postgres_objects.go

```diff
@@ -218,3 +228,41 @@ func postgresService(namespace string) *corev1.Service {
 
 	return service
 }
+
+// this is a pretty egregious hack to enable development against
+// clusters without storage classes ready. This is primarily for
+// delivering/managing CNI impls with kotsadm, since any non-cloud-specific
+// volume provisioner (e.g. rook/ceph, gluster) will probably require
+// a functioning pod network.
+//
+// Repeat: using a throwaway PV is really dangerous, but it enables
+// us to get kots up and running on a single node without a pod network.
```

Comment:
do we have a way to migrate from this throwaway pv to an actual storage provisioner following initial install?